README.md

```diff
@@ -270,29 +270,32 @@ Whenever a change in the internal graph occurs.
 
 ## API
 ### `compiler = bankai(entry, [opts])`
-Create a new bankai instance. Takes either an entry file location, or an array
-of files.
+Create a new bankai instance from the route to JavaScript file. The following
+options are available:
 
-### `compiler.documents(routename, [opts], done(err, buffer))`
+- __opts.quiet:__ Defaults to `false`. Don't output any data to `stdout`. Useful
```

Comment:
can we add `opts.watch` here too?

Reply:
Oh, is that an option? Yeah, we probably should then. Thought it wasn't tho :eyes: